docs/site/components/docs/search-overview.html

```diff
@@ -0,0 +1,7 @@
+<h3>Search Overview</h3>
+<p>
+  The object returned from <code>gcloud.search</code> gives you complete access to store and search your indexes and documents.
```

Comment:
Changed to "gives you complete access to store your documents and search your indexes."